tests/makeotfexe_data/input/bug993/feat.fea

```diff
@@ -1,5 +1,5 @@
 markClass [acute grave] <anchor 150 -10> @TOP_MARKS;                            
                                                                                 
-lookup mark {                                                                   
```

Comment:
Is `mark` not a valid lookup name now?

Reply:
This is presumably negotiable. `mark` is a keyword and has been special-cased as a valid tag. Labels are arbitrary strings that aren't output into generated fonts, so there's no reason for keyword/label overlap. However, if there is enough history it can be added back in.

Reply:
It is also a breaking change, and the fact that two tests already used `mark` as lookup names would be an indication that it is somehow a normal thing to do. I often name lookups with the feature name they will be used in, and it has always been accepted.

Reply:
@khaledhosny see https://github.com/adobe-type-tools/afdko/pull/1367#discussion_r646953030

Reply:
Good news! @iterumllc has kindly made an update (https://github.com/adobe-type-tools/afdko/pull/1367/commits/35f29a2b385f9696ffd072530758d535070179a2) that retains `mark` as a valid lookup name.